components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/NodeDependency.scala

```diff
@@ -89,7 +89,10 @@ case class Parameter(name: String,
                      branchParam: Boolean,
                      isLazyParameter: Boolean,
                      scalaOptionParameter: Boolean,
-                     javaOptionalParameter: Boolean) extends NodeDependency {
+                     javaOptionalParameter: Boolean,
+                     validationErrorMessage: Option[String] = None,
```

Comment:
It makes no sense. We can have many validators and each validator has it's own message.

Reply:
that's the user-defined one for the user-defined validator, I guess it could be named clearer

Reply:
or even bundled together with the CustomExpressionParameterValidator class